user_tools/tests/spark_rapids_tools_e2e/resources/scripts/hdfs/templates/core-site.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+  Copyright (c) 2024, NVIDIA CORPORATION.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<configuration>
+    <property>
+        <name>fs.defaultFS</name>
+        <value>hdfs://localhost:9000</value>
```

Comment:
what if I have an hdfs install already on my box?

Reply:
Incase a test requires HDFS, added a check in `test_steps.step_setup_hdfs()` to test if hdfs is active.
Incase there is an active HDFS running, the test fails with a warning saying:
```
HDFS is already active. Please stop it before running the tests.
``` 

We could setup HDFS by specifying a different port and http address but I would prefer to avoid it for now.